.github/workflows/validate_jsonschemas.yaml

```diff
@@ -13,17 +13,25 @@
 # limitations under the License.
 
 name: Validate JSON Schemas
-on: [push, pull_request]
+on:
+  pull_request:
+    paths:
+      - 'jsonschema/**'
+      - 'scripts/**'
+      - 'tools/**'
+      - '.github/workflows/validate_jsonschemas.yaml'
 
 jobs:
   validate:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
     - name: Check out repo
       uses: actions/checkout@v3
+
     - name: Setup Node
       uses: actions/setup-node@v3
       with:
         node-version: '12'
```

Comment:
I'm glad it was kept out of scope, but we should probably update the Node.js version to something supported.